sensor/common/connection/connection.go

```diff
@@ -0,0 +1,135 @@
+package connection
```

Comment:
Hm, not quite happy with the name as `connection` is very vague and the methods are called `GRPC` something.
Keep in mind Sensor also establishes connections to other services, can you make it Central specific?

You could move it to `centralclient` into a new `grpc` package.

Reply:
Changed the package and renamed some functions. Let me know if you think it's clearer now.

Reply:
I prefer the file to be moved to `grpc/connection.go` or call it `grpc_connection.go` for better searchability.

Reply:
Done